File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/template/pojo/Info.java
package com.yss.uco.elecreco.er.template.pojo;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;

/**
 * 模板基础信息
 * 
 * @author liuxiang
 * @date 2015年2月12日
 */
@XmlAccessorType(XmlAccessType.FIELD)
public class Info {
	/**
	 * 模板代码
	 */
	@XmlElement(name = "code")
	protected String code = "";

	/**
	 * 模板名称
	 */
	@XmlElement(name = "name")
	protected String name = "";

	/**
	 * 模板类型
	 */
	@XmlElement(name = "type")
	protected String type = "";

	/**
	 * 描述信息
	 */
	@XmlElement(name = "desc")
	protected String desc = "";

	/**
	 * 版本号
	 */
	@XmlElement(name = "version")
	protected String version = "";

	public String getCode() {
		return code;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getDesc() {
		return desc;
	}

	public void setDesc(String desc) {
		this.desc = desc;
	}

	public String getVersion() {
		return version;
	}

	public void setVersion(String version) {
		this.version = version;
	}
}

File: AMS_电子对账/code/client/source/YssElecReco/Pojo/Deploy/Info.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace YssElecReco.pojo.Deploy
{
    /// <summary>
    /// 模板基础信息
    /// </summary>
    public class Info
    {
        /// <summary>
        /// 模板代码
        /// </summary>
        private string code = "";

        /// <summary>
        /// 模板名称
        /// </summary>
        private string name = "";

        /// <summary>
        /// 模板类型
        /// </summary>
        private string type = "";

        /// <summary>
        /// 描述信息
        /// </summary>
        private string desc = "";

        /// <summary>
        /// 版本信息
        /// </summary>
        private string version = "";

        /// <summary>
        /// 模板代码
        /// </summary>
        [XmlElement("code")]
        public string Code
        {
            get { return code; }
            set { code = value; }
        }

        /// <summary>
        /// 模板名称
        /// </summary>
        [XmlElement("name")]
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        /// <summary>
        /// 模板类型
        /// </summary>
        [XmlElement("type")]
        public string Type
        {
            get { return type; }
            set { type = value; }
        }

        /// <summary>
        /// 描述信息
        /// </summary>
        [XmlElement("desc")]
        public string Desc
        {
            get { return desc; }
            set { desc = value; }
        }

        /// <summary>
        /// 版本信息
        /// </summary>
        [XmlElement("version")]
        public string Version
        {
            get { return version; }
            set { version = value; }
        }
    }
}
